src/python/finalize/finalize.go

```diff
@@ -96,6 +96,14 @@ func (f *Finalizer) HandleCollectstatic() error {
 	return nil
 }
 
+func hasDjangoAsDependency(f *Finalizer) error {
+	if err := f.Command.Execute(f.Stager.BuildDir(), os.Stdout, os.Stderr, "pip-grep", "-s", "requirements.txt", "django", "Django"); err != nil {
+		return f.Command.Execute(f.Stager.BuildDir(), os.Stdout, os.Stderr, "pip-grep", "-s", "Pipfile", "django", "Django")
```

Comment:
`pip-grep -s Pipfile django Django` is throwing an exception. It appears that pip-grep only has support for requirements.txt, not PIpfiles. Can you find another way to check the existence of Django?